app/upgrades/v25/constants.go

```diff
@@ -1,23 +1,42 @@
 package v25
 
 import (
+	"github.com/osmosis-labs/osmosis/osmomath"
 	"github.com/osmosis-labs/osmosis/v24/app/upgrades"
 
 	store "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	auctiontypes "github.com/skip-mev/block-sdk/x/auction/types"
 
 	smartaccounttypes "github.com/osmosis-labs/osmosis/v24/x/smart-account/types"
 )
 
 // UpgradeName defines the on-chain upgrade name for the Osmosis v25 upgrade.
-const UpgradeName = "v25"
+const (
+	UpgradeName = "v25"
+
+	// Noble USDC is used as the auction denom
+	AuctionUSDCDenom = "ibc/498A0751C798A0D9A389AA3691123DADA57DAA4FE165D5C75894505B876BA6E4"
+)
 
 var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
 	StoreUpgrades: store.StoreUpgrades{
 		Added: []string{
+			auctiontypes.StoreKey,
 			smartaccounttypes.StoreKey,
 		},
 		Deleted: []string{},
 	},
 }
+
+// AuctionParams expected initial params for the block-sdk
+var AuctionParams = auctiontypes.Params{
+	MaxBundleSize:          5,
+	ReserveFee:             sdk.NewCoin(AuctionUSDCDenom, sdk.NewInt(1000000)),
+	MinBidIncrement:        sdk.NewCoin(AuctionUSDCDenom, sdk.NewInt(1000000)),
+	EscrowAccountAddress:   auctiontypes.DefaultEscrowAccountAddress,
+	FrontRunningProtection: true,
+	ProposerFee:            osmomath.MustNewDecFromStr("0.05"),
```

Comment:
How do the funds actually go to the proposer? Does it get sent to the validator's stakers? To the validators personal operator address?

Reply:
In the auction ante handler => https://github.com/skip-mev/block-sdk/blob/v1.4.2/x/auction/ante/ante.go#L73-L75

The ValidateBidInfo on the auctionkeeper which calls ValidateAuctionBid:
https://github.com/skip-mev/block-sdk/blob/v1.4.2/x/auction/keeper/auction.go#L24-L26

ValidateAuctionBid calls ExtractBid here:
https://github.com/skip-mev/block-sdk/blob/v1.4.2/x/auction/keeper/auction.go#L91

Which sends funds to the escrow account, sans the proposer fee:
https://github.com/skip-mev/block-sdk/blob/v1.4.2/x/auction/keeper/auction.go#L91

Which will send the percent of funds to the rewardsAddress => https://github.com/skip-mev/block-sdk/blob/v1.4.2/x/auction/keeper/auction.go#L110

And the rewards address is defined here in a function:
https://github.com/skip-mev/block-sdk/blob/main/x/auction/rewards/proposer_provider.go#L29-L41

Reply:
So its the operator address, gotcha